app.yaml

```diff
@@ -40,9 +40,17 @@ handlers:
     # WARNING TO DEVELOPERS: Files in this folder may be stale for
     # up to 10 mins after a code release.
     Cache-Control: 'public, max-age=600'
+- url: /assets/(constants\.json)
```

Comment:
Re the additional rule, I'm still not sure I get it. In production, shouldn't the frontend be making a request to build/, not to assets/constants.json? You don't need a longer max-age during development...

Also, I talked to @BenHenning about the other issue, and you raise a good point. We both think it's fine to keep a single GLOBAL for ASSET_DIR_PREFIX. However, it would be nice to move that into a separate JS file and auto-generate that file (or sed replace, or something) each time a new release happens; we'd also need to think about how to handle that with dev environments. I think it's fine to do this in a separate PR; if you agree, would you mind filing an issue for this?

Thanks!